integrations/Vero/browser.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable class-methods-use-this */
+import get from "get-value";
+import { NAME } from "./constants";
+import logger from "../../utils/logUtil";
+
+class Vero {
+  constructor(config) {
+    this.apiKey = config.apiKey;
+    this.name = NAME;
+  }
+
+  init() {
+    logger.debug("===In init Vero===");
+    window._veroq = window._veroq || [];
+    (function () {
+      var ve = document.createElement("script");
+      ve.type = "text/javascript";
+      ve.async = true;
+      ve.src = "https://d3qxef4rp70elm.cloudfront.net/m.js";
+      var s = document.getElementsByTagName("script")[0];
+      s.parentNode.insertBefore(ve, s);
+    })();
```

Comment:
Use the ScriptLoader utility.
CC @utsabc

Reply:
Done